recipe/build.sh

```diff
@@ -14,7 +14,10 @@ fi
 
 # Install Cyclus
 #export VERBOSE=1
-${PYTHON} install.py --prefix="${PREFIX}" \
+${PYTHON} install.py \
+  --prefix="${PREFIX}" \
+  --cmake_prefix_path="${PREFIX}" \
+  --deps-root="${PREFIX}" \
```

Comment:
@conda-forge/cyclus, I've taken a shot a cleaning this up a bit, but I'm running into problems here. The cyclus build does not in fact respect `CMAKE_PREFIX_PATH`, nor `LDFLAGS` (which for us contains `-L$PREFIX/lib`), otherwise it would not run into missing symbols from libraries that are plainly present there.

I've stared at the [upstream](https://github.com/cyclus/cyclus/blob/main/install.py) [build](https://github.com/cyclus/cyclus/blob/main/CMakeLists.txt) for a while, but cannot figure out where this goes wrong. Any thoughts?